api/core/api.proto

```diff
@@ -114,13 +118,15 @@ message ResultData {
 // {
 //   "serviceID": "__SERVICE_ID__",
 //   "taskKey":   "__TASK_KEY__",
-//   "inputData": "{\"foo\":\"bar\"}"
+//   "inputData": "{\"foo\":\"bar\"}",
+//   "tags":      ["executionX", "test"]
 // }
 // ```
 message ExecuteTaskRequest {
-  string serviceID = 1; // The Service ID. Generated when using the [`DeployService` API](#deployservice).
-  string taskKey = 2;   // The task's key to execute.
-  string inputData = 3; // The inputs of the task to execute, encoded in JSON.
+  string serviceID = 1;     // The Service ID. Generated when using the [`DeployService` API](#deployservice).
+  string taskKey = 2;       // The task's key to execute.
+  string inputData = 3;     // The inputs of the task to execute, encoded in JSON.
+  repeated string tags = 4; // List of tags to assign to this task
```

Comment:
Again, This is not the task's tags but the execution's tags.

Should be updated to "The list of tags to associate to the execution"

Reply:
Same here about renaming `tags` to `executionTags`.

Reply:
the same here I feel `tags` is better because we are already in an execution `ExecuteTaskRequest`

Reply:
Let's continue the conversation on the other comment ;) It's the same subject.